pkgs/development/haskell-modules/configuration-hackage2nix.yaml

```diff
@@ -3637,8 +3637,6 @@ broken-packages:
   - cmt
   - cmv
   - cnc-spec-compiler
-  - co-log
```

Comment:
Btw, newer updated versions of `co-log-core` and `co-log` are released to Hackage:

* http://hackage.haskell.org/package/co-log-core-0.2.1.1
* http://hackage.haskell.org/package/co-log-0.4.0.1